packages/nimble-components/src/card/specs/README.md

```diff
@@ -0,0 +1,111 @@
+# Nimble Card
+
+## Overview
+
+The `nimble-card` is a container that is designed to contain arbitrary content specified by a client
+application. The `nimble-card` is intended for grouping related content.
+
+### Background
+
+[GitHub Issue #296: nimble-card component](https://github.com/ni/nimble/issues/296)
+
+An IxD and Visual Design spec for the `nimble-card` component have not yet been provided. We will add some
+preliminary styling in the interim.
+
+The new Routines UX in SystemLink Enterprise calls for grouping related configuration in a slide-out in labeled
+containers. See [Figma: NorthStar Vision for SLE Routines](https://www.figma.com/file/VQ7WIi3qqRG3r19VXqVvem/Stratus-Routines?type=design&node-id=1%3A74129&mode=design&t=KfPTwy4IhHbt42LR-1).
+We are designing the `nimble-card` component to address that use case.
+
+---
+
+## Design
+
+The `nimble-card` will not include any built-in content (title, footer, actions, etc.). Some component libraries
+do provide a card component with built-in, pre-styled content or pre-styled child components to use inside the card
+component. See the [Angular Material `mat-card` component](https://v5.material.angular.io/components/card/overview)
+for an example. We could add similar content in the future, but it is not required at this time.
```

Comment:
It's interesting that you call having additional built-in content as not required at this time, but your code example includes a header (via `h2` element) and the figma document linked has headers in each section. I'm leaning towards saying we should have a "title" slot that makes it easy to put a title in the section with the correct font, size, color, spacing, etc.

That suggestion does require creating our own copy of the component's template, but in this case, I think the deviation from what FAST provides is probably worth it in order to help enforce consistency across usages. However, since we don't have designs or UX guidance on how the card would be used more broadly, maybe it's better to track this in the "Open Issues" section for now. That'll give other reviewers/owners a chance to weigh in.

Reply:
The h2 in the example is just because I was copying the fast-card's example content in the explorer: https://explore.fast.design/components/fast-card

I would also lean toward adding a title, though. I think it makes sense and adds a lot of value to the card. In the initial design conversations, Mark and Katie suggested that we not have it because there might be cases where we'd want to use the card without a title. Seems like it would be easy enough to support that possibility by making the title optional, though.

I'll add it to the Open Issues for now, to have more discussion.

Reply:
That sounds good. We have some components (dialog and banner at least) that have a title that can optionally be hidden. Depending on where we land on the ARIA role, being able to set a title, even if it isn't visually shown, will be important to make sure the card as an accessible label.

Reply:
I added an item under open issues about this question.

I also updated the [prototype](https://github.com/ni/nimble/tree/users/kelseyj/card-prototype) to have a title slot (no title-hidden option, I'll save that for if we decide to do it).

Reply:
I agree that an optional title seems useful.

Reply:
I updated the spec to say that we will include an optional title. Since we're including a title, I assume we also want to set `aria-label` to that title and and add a `title-hidden` attribute (similar to nimble-banner), so I added those to the spec as well.

Reply:
It doesn't sound like we're giving the `nimble-card` an ARIA role, so we don't actually have anything to label with the title (no role on the card means that it won't be part of the accessibility tree, and ARIA labels provide a label for items in the accessibility tree).

Reply:
Is it semantically like a section? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

Reply:
Ok, that makes sense. I removed the note about a label and the title-hidden/accessibly-hidden details, based on our conversation.

Reply:
I think we would still need title-hidden (I made an update to my comment so maybe that was missed)
i.e. visually if you don't want the header visible. Or are you saying with a card the header is always visible?

Reply:
Sorry, my "that makes sense" was to Molly -- I hadn't refreshed the page to see your comment. The idea behind removing "title-hidden" was that, without the need to set the accessibility attributes, we can achieve collapsing/hiding the title completely without an attribute for it, if there is no content for the title slot.

Reply:
Think I'm advocating for it to show up in the accessibility tree as a section. If that's the direction, we would want users to always put something in the title slot so some context is given to the accessibility tree of this visual group of stuff (even if visually the title is hidden).

Reply:
I updated the spec again, to include the title slot. After further discussion, we don't want to include the title-hidden property for now. We don't intend to provide styling for the no-title case and it's possible the designers will say it always needs a title in the end, so we can wait to provide the option to hide the title until we have an actual use case for it.